src/test/persistedEntities.ts

```diff
@@ -12,5 +12,5 @@ export async function persistedUserWithRoleAndPermissions(roleName: string, perm
 
   await Promise.all(permissionNames.map((permissionName) => permissionRepository.save(new Permission(permissionName))));
   await roleRepository.save(role);
-  return userRepository.save(admin);
+  return await userRepository.save(admin);
```

Comment:
```suggestion
  return userRepository.save(admin);
```
Not needed, as it returns a promise anyhow